### PR TITLE
Fix typos in readme

### PR DESCRIPTION
**Pull request by mtsknn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ $string = str('john pinkerton');
 ```
 
 Once you have a concrete `Twine\Str` instance you may treat it like any other
-string. This includes echoing it or using any of PHP's built in string functions
+string. This includes echoing it or using any of PHP's built-in string functions
```
Comment (PHLAK) [inline README.md:71]: "[built-in](https://www.merriam-webster.com/dictionary/built-in)" is correct.

File: README.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ $string->snakeCase(); // Returns 'john_pinkerton'
 ```
 
 At this point you're ready to start using Twine by calling any of its many
-built in methods.
+built-in methods.
```
Comment (PHLAK) [inline README.md:98]: Same as above.

**Review by PHLAK — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mtsknn) [reply README.md:71]: Yes, and that's what I used. Also, "built-in" is already used on line 82.

Comment (PHLAK) [reply README.md:71]: Oh sorry, I read that backwards for some reason, thought you were removing the dash (maybe I shouldn't review PRs in the middle of the night).

Comment (mtsknn) [reply README.md:71]: 😄 No worries. Get some sleep!

**Pull request merged**